1-org/README.md

```diff
@@ -163,6 +163,15 @@ Run `terraform output cloudbuild_project_id` in the `0-bootstrap` folder to see
    if [ ! -z "${ACCESS_CONTEXT_MANAGER_ID}" ]; then sed -i "s=//create_access_context_manager_access_policy=create_access_context_manager_access_policy=" ./envs/shared/terraform.tfvars; fi
    ```
 
+1. **(Optional)** If you are deploying The Terraform Example Foundation in a Folder instead of an Organization, you maybe want to add a suffix in tag keys to avoid collision. The script below enables a terraform variable to achieve this:
+
+   ```bash
+   export PARENT_FOLDER=$(terraform -chdir="../terraform-example-foundation/0-bootstrap/" output -json common_config | jq '.parent_folder' --raw-output)
+   echo "PARENT_FOLDER = $PARENT_FOLDER"
+
+   if [ ! -z "${PARENT_FOLDER}" ]; then sed -i "s=//create_unique_tag_key=create_unique_tag_key=" ./envs/shared/terraform.tfvars; fi
+   ```
+
```

Comment:
Could these instructions also be added to the  instructions for?
- Deploying with Jenkins
- Running Terraform locally

Reply:
Done